tracer/build/PackageVersionsGeneratorDefinitions.json

```diff
@@ -425,5 +425,16 @@
       "MaxVersionExclusive": "2.37.0",
       "SkipArm64": true
     }]
+  },
+  {
+    "IntegrationName": "OpenTelemetry",
```

Comment:
Actually... I was reviewing one last time Zach's last [PR](https://github.com/DataDog/dd-trace-dotnet/pull/3572), and he adds the `OpenTelemetry` package with the same Integration Name. I assume we can't have 2 package with the same IntegrationName, can we?

Reply:
> I assume we can't have 2 package with the same IntegrationName, can we?

🤷 I guess we'll find out! 😄